openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/OpenShiftAssistantTemplate.java

```diff
@@ -0,0 +1,72 @@
+package org.arquillian.cube.openshift.impl.client;
+
+import io.fabric8.kubernetes.api.model.v3_1.KubernetesList;
+import io.fabric8.kubernetes.clnt.v3_1.KubernetesClient;
+import io.fabric8.openshift.api.model.v3_1.DoneableTemplate;
+import io.fabric8.openshift.api.model.v3_1.Template;
+import io.fabric8.openshift.clnt.v3_1.OpenShiftClient;
+import io.fabric8.openshift.clnt.v3_1.ParameterValue;
+import io.fabric8.openshift.clnt.v3_1.dsl.TemplateResource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OpenShiftAssistantTemplate {
+
+    private final KubernetesClient client;
```

Comment:
Can we change this to `OpenShiftClient`, in `OpenShiftAssistant` it uses the openshift client so you can pass it directly and you save later (line 54) the `adapt` call